actionpack/lib/action_controller/base.rb

```diff
@@ -267,6 +267,11 @@ def _protected_ivars
     end
     private :_protected_ivars
 
+    class << self
+      attr_accessor :silence_disabled_session_errors
+    end
+    @silence_disabled_session_errors = true
+
```

Comment:
Why here and not a `config_accessor` in `RequestForgeryProtection`?

Reply:
Because that's where configs are stored by default. It also avoid eager loading `RequestForgeryProtection` if it's not used (at least I think, I admit I haven't double checked)

Reply:
Sorry, I wasn't being clear - we have an `included` block in `RequestForgeryProtection` here: 

https://github.com/rails/rails/blob/main/actionpack/lib/action_controller/metal/request_forgery_protection.rb#L66-L67

Since we're accessing the config in that module I think it's safe to assume that it's loaded at that point is it not?

Do we also need to have a block in railties configuration for setting the default in new apps vs. upgrading apps?

Reply:
> Since we're accessing the config in that module I think it's safe to assume that it's loaded at that point is it not?

Hum, I don't think so. Because if you have `load_defaults 7.0`, then we set `config.action_controller.silence...`, which will cause the action controller Railtie to try to set `ActionController::Base.silence...`.

> Do we also need to have a block in railties configuration for setting the default in new apps vs. upgrading apps?

Right here: https://github.com/rails/rails/pull/42231/files#diff-9f26e965e7e2a6842bd6ea755924a621038bc9dd8d63cfa6b34092aa32ede02cR204-R206 no?

Reply:
> Hum, I don't think so. Because if you have `load_defaults 7.0`, then we set `config.action_controller.silence...`, which will cause the action controller Railtie to try to set `ActionController::Base.silence...`.

My concern is that it's not a pattern that's used elsewhere in Action Pack and it's not in the module where it's used. There's a similar scenario with `default_protect_from_forgery` is there not? That's set in the `load_defaults 5.2` block so something prevents that from blowing up. The `default_protect_from_forgery` config is applied using the `:action_controller_base` load hook so could we not use the same approach?

> Right here: https://github.com/rails/rails/pull/42231/files#diff-9f26e965e7e2a6842bd6ea755924a621038bc9dd8d63cfa6b34092aa32ede02cR204-R206 no?

Sorry, obviously my 👀 are deteriorating with my advancing years 👴🏻

Reply:
> could we not use the same approach?

You are totally right, I missed this. I updated the PR.

> obviously my 👀 are deteriorating with my advancing years 👴🏻

🤣